app/models/user_role.rb

```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
 class UserRole < ActiveRecord::Base
   enum role: { admin: 0 }
+
+  belongs_to :gr_ministry, foreign_key: :ministry, class_name: 'Ministry', primary_key: :gr_id, inverse_of: :user_roles
```

Comment:
Name makes sense to me. The UserRole table wasn't designed well to link into active model relationships, so naming will be hard. It was designed more to allow someone (admin, non programmer maybe) to just insert new rows.